frontend/src/plugins/impl/data-frames/DataFramePlugin.tsx

```diff
@@ -109,6 +113,11 @@ export const DataFrameComponent = ({
           <CodePanel dataframeName={dataframeName} transforms={value} />
         </TabsContent>
       </Tabs>
+      {error && (
+        <div className="text-error border-[var(--red-6)] bg-[var(--red-2)] text-sm p-2 border">
+          {prettyError(error)}
+        </div>
+      )}
```

Comment:
I don't see this being rendered when I enter an invalid transform.

![image](https://github.com/marimo-team/marimo/assets/1994308/9a5329da-bf32-43de-95e7-dbcc60b6378a)